cityofaustin/atd-moped
# Drop partner name moped proj partners 10607

## Associated issues
fixes https://github.com/cityofaustin/atd-data-tech/issues/10607

## Testing
**URL to test:** 


https://drop_partner_name_moped_proj_partners_10607--atd-moped-main.netlify.app/moped/

**Steps to test:**

go to a project, add or edit the project partners and save. the ui should reflect the changes.

---
#### Ship list
